components/playback/controls/BarcodeViewer.vue

```diff
@@ -137,6 +137,7 @@ export default {
         type: this.barcode_type,
         new_value: event.target.value,
       });
+      console.log("Barcode Set Manually"); // allow-log
```

Comment:
The way this works is that once you enable manual entry mode, you cannot disable it, so I think we can just add this log entry to `handle_manual_mode_choice` above, but make sure that it's actually being set to manual mode using the `choice` arg